script/clone_job.pl

```diff
@@ -113,7 +113,7 @@ =head1 SYNOPSIS
 use JSON;
 
 use FindBin;
-use lib "$FindBin::Bin/../lib";
+BEGIN { unshift @INC, "$FindBin::RealBin/../lib" }
 use OpenQA::Client;
```

Comment:
I am just using the same approach as in script/client . Any better way to do that applies for all? https://build.opensuse.org/request/show/517452 is the use case. I want to symlink these scripts like openqa-client so that one can install just the openSUSE package openQA-client and has also openqa-clone-job in PATH. For that one needs to resolve the absolute path (RealBin) to be able to find OpenQA::Client, right?